app/helpers/sellers_helper.rb

```diff
@@ -5,36 +5,114 @@ def self.generate_seller_json(seller:)
     locations = seller.locations
     seller = seller.as_json
     seller['locations'] = locations.as_json
-    seller['amount_raised'] = SellersHelper.calculate_amount_raised(
+    gift_card_amount = SellersHelper.calculate_gift_card_amount(
       seller_id: seller['id']
     )
+    donation_amount = SellersHelper.calculate_donation_amount(
+      seller_id: seller['id']
+    )
+    seller['gift_card_amount'] = gift_card_amount
+    seller['donation_amount'] = donation_amount
+    seller['amount_raised'] = gift_card_amount + donation_amount
+    num_gift_cards = SellersHelper.calculate_num_gift_cards(
```

Comment:
I think it's cleaner if you just assign it like `seller['gift_card_amount']` and invoke the method you're calling up there